website/docs/getting-started/setup/installation-guides/docker/README.mdx

```diff
@@ -1,203 +1,84 @@
 ---
-description: Appsmith can be deployed locally or on a private instance using Docker
-sidebar_position: 1
+description: Deploy Appsmith locally or on a private instance with ease using Docker Compose. Follow these simple steps to set up Appsmith on your Docker Compose platform.
 ---
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
 # Docker
 
-Docker is an open source [containerization](https://www.ibm.com/in-en/cloud/learn/containerization) platform. It enables developers to package applications into standardized executable components called containers. These containers combine the application's source code with the Operating System (OS) libraries and dependencies required to run that code in any environment.
+This page provides stepwise instructions to help you install Appsmith using Docker.
 
-<VideoEmbed host="youtube" videoId="Tde7GqE6FQQ" /> 
+<VideoEmbed host="youtube" videoId="Tde7GqE6FQQ" title="How to install Appsmith using Docker" 
+caption="How to install Appsmith using Docker"/> 
 
 ## Prerequisites
 
-* [Docker](https://docs.docker.com/get-docker/) (version 20.10.7 or later)
-* [Docker-Compose](https://docs.docker.com/compose/install/) (version 1.29.2 or later)
+1. [Docker](https://docs.docker.com/get-docker/) (version 20.10.7 or later)
+2. [Docker-Compose](https://docs.docker.com/compose/install/) (version 1.29.2 or later)
+3. Create a folder named `appsmith` for deployment and data storage, then navigate to it using `cd` command.
```

Comment:
Put point no 3 as the first point under Install Appsmith section. It makes more sense there

Reply:
I changed this a bit.